agnpy/wrappers/gammapy.py

```diff
@@ -0,0 +1,208 @@
+from multiprocessing.sharedctypes import Value
+from xml.etree.ElementInclude import XINCLUDE_INCLUDE
+import astropy.units as u
+from ..synchrotron import Synchrotron
+from ..compton import ExternalCompton
+from ..compton import SynchrotronSelfCompton
+from gammapy.modeling.models import SpectralModel, Parameter, Parameters
+
+
+class SpectralModelSSC(SpectralModel):
+
+    tag = ["SpectralModelSSC"]
+
+    def __init__(self, blob):
+        """Initialise all the parameters from the Blob instance."""
+
+        self.blob = blob
+
+        parameters = []
+
+        # all the parameters of the EED have to be parameters of the fittable model
+        # any electron distribution has for attributes the spectral parameters and the integrator
+        # we remove the latter
+        pars = vars(self.blob.n_e)
+        pars.pop("integrator")
+        for name in pars.keys():
+            parameter = Parameter(name, pars[name])
+            parameters.append(parameter)
+
+        # emission region parameters
+        z = Parameter("z", blob.z)
+        d_L = Parameter("d_L", blob.d_L)
+        delta_D = Parameter("delta_D", blob.delta_D)
+        B = Parameter("B", blob.B)
+        R_b = Parameter("R_b", blob.R_b)
+        parameters.extend([z, d_L, delta_D, B, R_b])
+
+        self.default_parameters = Parameters(parameters)
```

Comment:
gammapy models parameters are usually defined at class level and come with default values, but I guess you prefer to avoid this because it's hard to define good defaults here.

Reply:
Yes, it is a bit difficult to establish defaults, in general. Additionally each emission region comes with its own electron distribution of which I do not know the parameters. I added functions to fetch these parameters from the physical objects and set some reasonable ranges of values.